addons/mail/static/src/components/emoji_grid_loading_screen/emoji_grid_loading_screen.js

```diff
@@ -0,0 +1,21 @@
+/** @odoo-module **/
+
+import { registerMessagingComponent } from '@mail/utils/messaging_component';
+
+const { Component } = owl;
+
+export class EmojiGridLoadingScreen extends Component {
+    /**
+     * @returns {EmojiGridLoadingScreen}
+     */
+    get EmojiGridLoadingScreen() {
```

Comment:
note as I see it, this should have been lowercase `e` at the start

Reply:
Oops, good catch. My mistake. Don't think it's worth a `[FIX]` PR for that though